src/controllers/time.rs

```diff
@@ -53,6 +53,7 @@ impl TimeController {
     /// `dt` is the amount of seconds that have passed since the last update
     pub fn update_seconds(&mut self, dt: f64, actions: &Actions, state: &mut GameState) {
         self.current_time += dt;
+        state.difficulty += 0.001;
```

Comment:
The increase in difficulty should depend on `dt`, which is the elapsed time since the last call to `update_seconds`. Otherwise the speed will increase faster when the game runs at a higher update rate.

Reply:
Good point, now it uses `dt`.